geode-core/src/main/resources/META-INF/schemas/geode.apache.org/schema/cache/cache-1.0.xsd

```diff
@@ -205,7 +205,7 @@ declarative caching XML file elements unless indicated otherwise.
           </xsd:complexType>
         </xsd:element>
 
-        <xsd:element maxOccurs="unbounded" minOccurs="0" name="gateway-receiver">
+        <xsd:element maxOccurs="1" minOccurs="0" name="gateway-receiver">
```

Comment:
I don't think any customer is configuring multiple gateway receivers per node because it would throw an mbean exception, but are we allowed to just change the xsd like this? Do we need to 'deprecate' multiple receivers per node?

Reply:
Hello @boglesby,

As you said, I don't think anybody is actually trying to configure more than one `gateway-receiver` per node, the member will just throw an exception and shutdown when trying to register the `MBean`. That's why I added the validation int he DTD/XSD as well, primarily to fail fast and allow earlier detection of the issue for people using XML modeling tools and/or editors.
Just for the record, I was also concerned about "breaking the backward compatibility" by modifying these files, so I had a discussion with @jhuynh1 / @nabarunnag and they agreed with the change. Bottom line, even if we don't change the DTD/XSD the member will fail when configuring more than one `gateway-receiver` so we're not actually breaking the backward compatibility, but just making sure the product is configured as it should.
Just my two cents, let me know what you think.
Cheers.